controller/lib/core/sensors.cpp

```diff
@@ -181,42 +181,49 @@ void TVIntegrator::AddFlow(Time now, VolumetricFlow flow) {
   }
 }
 
-SensorReadings Sensors::GetSensorReadings() {
-  // Flow rate is inhalation flow minus exhalation flow. Positive value is flow
-  // into lungs, and negative is flow out of lungs.
+std::pair<SensorValues, SensorReadings> Sensors::GetSensorReadings() {
   auto now = Hal.now();
-  auto patient_pressure = ReadPressureSensor(PATIENT_PRESSURE);
-  auto inflow_delta = ReadPressureSensor(INFLOW_PRESSURE_DIFF);
-  auto outflow_delta = ReadPressureSensor(OUTFLOW_PRESSURE_DIFF);
-
-  VolumetricFlow inflow = PressureDeltaToFlow(inflow_delta);
-  VolumetricFlow outflow = PressureDeltaToFlow(outflow_delta);
-  VolumetricFlow uncorrected_flow = inflow - outflow;
-  VolumetricFlow corrected_flow = uncorrected_flow + flow_correction_;
+
+  SensorValues values;
```

Comment:
My preference would be to avoid default-constructing the SensorValues struct.  In fact, I think we should delete its default constructor so you can't do this:

```
struct SensorValues {
  SensorValues() = delete;
};
```

The reason is, if we add a field to the SensorValues struct, we want to make sure it's always filled in.  Allowing default initialization lets us break that invariant and leave some fields uninitialized.

You can just construct the struct at return time the same way we used to, `return {.foo = bar, ...}`.  I like these "dotted initializers" because it makes it harder to e.g. swap two pressure fields.